lib/perl/Genome/Model/Tools/DetectVariants2/Lumpy.pm

```diff
@@ -17,7 +17,12 @@ class Genome::Model::Tools::DetectVariants2::Lumpy {
             calculate_from => ['_temp_staging_directory'],
             calculate => q{ File::Spec->join($_temp_staging_directory, 'legend.tsv'); },
         },
-    ]
+    ],
+    has_param => [
+        lsf_resource => {
+            default_value => "-M 16000000 -R 'select[mem>16000] rusage[mem=16000]'",
```

Comment:
Yeah, my test build ran into that problem!  I am also going to investigate whether we can use StreamGraph to optimize this a little bit and not write out some of the files.  But we will still need temp space.